Dockerfile

```diff
@@ -42,28 +42,28 @@ COPY dist/$TARGETOS/$TARGETARCH/$BIN_NAME /bin/
 # storage backend, if desired; the server will be started with /vault/config as
 # the configuration directory so you can add additional config files in that
 # location.
-RUN mkdir -p /vault/logs && \
-    mkdir -p /vault/file && \
-    mkdir -p /vault/config && \
-    chown -R ${NAME}:${NAME} /vault
+RUN mkdir -p /openbao/logs && \
+    mkdir -p /openbao/file && \
+    mkdir -p /openbao/config && \
+    chown -R ${NAME}:${NAME} /openbao
 
 # Expose the logs directory as a volume since there's potentially long-running
 # state in there
-VOLUME /vault/logs
+VOLUME /openbao/logs
 
 # Expose the file directory as a volume since there's potentially long-running
 # state in there
-VOLUME /vault/file
+VOLUME /openbao/file
 
 # 8200/tcp is the primary interface that applications use to interact with
-# Vault.
+# OpenBao.
 EXPOSE 8200
 
 # The entry point script uses dumb-init as the top-level process to reap any
-# zombie processes created by Vault sub-processes.
+# zombie processes created by OpenBao sub-processes.
 #
 # For production derivatives of this container, you shoud add the IPC_LOCK
-# capability so that Vault can mlock memory.
+# capability so that OpenBao can mlock memory.
 COPY .release/docker/docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
 ENTRYPOINT ["docker-entrypoint.sh"]
```

Comment:
AFAICT, `docker-entrypoint.sh` isn't modified in this PR, so I don't think this will work. Are you interested in doing that? :-)